prepare_check_cleanup/migration.py

```diff
@@ -36,7 +36,7 @@ class Action(str, Enum):
         PREPARE: Prepare discourse pages before running the migration.
         CHECK_BRANCH: Check that the migration branch was created.
         CHECK_PULL_REQUEST: Check that the migration pull request was created.
-        CLEANUP: Delete discourse pages before after the migration.
+        CLEANUP: Delete discourse pages and migration pull request and branch after the migration.
```

Comment:
I would change to `pages, migration pull request and branch` to avoid repeated `and`

Reply:
the second `and` is there to highlight that the `migration` modifier to `pull request` also applies to `branch`, writing it as `pages, migration pull request and branch` can be read that the `migration` modifier only applies to the `pull request` and not `branch`